libs/common/src/constants/index.ts

```diff
@@ -24,6 +24,5 @@ export * from './url-character-allowances';
 export * from './max-character-count';
 export * from './deal-submission-type';
 export * from './floating-point-rounding-decimal-places';
-export * from './cancellation-status';
```

Comment:
I moved the deal cancellation status constants inside the `tfm` folder which is why these changes are showing up

Reply:
Good spot - thanks! Tbf the name should probably be tfm-cancellation-status too oops

Reply:
Have updated the name!